src/server/cluster/cluster_family.cc

```diff
@@ -447,9 +447,6 @@ void ClusterFamily::Cluster(CmdArgList args, const CommandContext& cmd_cntx) {
 }
 
 void ClusterFamily::ReadOnly(CmdArgList args, const CommandContext& cmd_cntx) {
-  if (!IsClusterEmulated()) {
```

Comment:
please open an issue to actually implement this behavior. it can be minor for now.